.github/workflows/test.yml

```diff
@@ -38,8 +38,3 @@ jobs:
         run: |
           forge build --sizes
         id: build
-
-      - name: Run Forge tests
-        run: |
-          forge test -vvv
-        id: test
```

Comment:
there were no tests previously and this was just failing the build for no reason